storage/container_storage_interface/persistent-storage-csi-manila.adoc

```diff
@@ -12,17 +12,18 @@ toc::[]
 Familiarity with link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVs], link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims[persistent volume claims (PVCs)], link:http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html[dynamic
 provisioning], and link:https://kubernetes.io/docs/admin/authorization/rbac/[RBAC authorization] is recommended.
 
-Before PVCs can be created, you must install the Manila CSI Driver Operator. The Operator creates the required storage classes for all available Manila share types needed for dynamic provisioning.
+To create CSI-provisioned PVs that mount to Manila storage assets, {product-title} installs the Manila CSI Driver Operator and the Manila CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
```

Comment:
@jsafrane Do we need to suggest that the user upgrades the old Operator (if installed) before upgrading a cluster to allow an easier namespace change? (This was mentioned during design, though I might be misunderstanding the purpose and placement.)

Reply:
Manila CSI Driver Operator and Manila CSI driver will be installed by default only when the OSP cluster has manila service enabled.

Manila CSI Driver Operator will be installed in "openshift-cluster-csi-drivers", but Manila CSI driver is installed in "openshift-manila-csi-driver."

If the user has installed the Manila CSI Driver Operator and Manila CSI driver with OLM in OCP 4.5,  we will ensure the upgrade to 4.6 success, no need user upgrades the old operator first.